fairlearn/metrics/_disparities.py

```diff
@@ -7,6 +7,46 @@
 from ._metric_frame import MetricFrame
 
 
+def average_odds_difference(
```

Comment:
Should there be an average odds ratio?

Reply:
Yeah, probably. I added one and corrected some typos related to the `method` argument in the previous ratio functions.